priv/repo/migrations/20241004172259_create_tiers.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Safira.Repo.Migrations.CreateTiers do
+  use Ecto.Migration
+
+  def change do
+    create table(:tiers, primary_key: false) do
+      add :id, :binary_id, primary_key: true
+      add :name, :string, null: false
+      add :priority, :integer, null: false
```

Comment:
```suggestion
      add :priority, :integer, null: false
```

Priority needs to be unique.

Reply:
This will not be possible due to the way tier priority is saved when moving them.